packages/noya-state/src/reducers/styleReducer.ts

```diff
@@ -301,6 +319,13 @@ export function styleReducer(
         if (!draft.fills[index].gradient) {
           draft.fills[index].gradient = Models.fill.gradient;
         }
+        if (!draft.fills[index].image) {
```

Comment:
Maybe we should only do this if the `type` is pattern, just so we don't create objects we don't need in the save data. Same with gradient above